examples/distributed_data_classification_examples/test_models.ipynb

```diff
@@ -0,0 +1,373 @@
+{
```

Comment:
I can move this file if we don't like having notebooks but personally i think for examples notebooks are a better way to understand.

Reply:
Actually while we're moving stuff around, do you mind also moving everything in `distributed_data_classification_examples` to the root `examples` folder? I think they should be with everything else. And rename the files too please.
- `domain_api_example.py` -> `domain_classifier_example.py`
- `quality_api_example.py` -> `quality_classifier_example.py`